### PR TITLE
clarify restrictions on imagestream tag aliases

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@soltysh @deads2k ptal.

**Comment by deads2k** [comment]
/lgtm

@bparees not sure if you want to wait for another review.

**Comment by deads2k** [comment]
accidental close, sorry

**Comment by bparees** [comment]
@deads2k yours is good enough for me :)

File: image/v1/generated.proto [hunk 1]
```diff
@@ -469,7 +469,8 @@ message TagReference {
   map<string, string> annotations = 2;
 
   // Optional; if specified, a reference to another image that this tag should point to. Valid values
-  // are ImageStreamTag, ImageStreamImage, and DockerImage.
+  // are ImageStreamTag, ImageStreamImage, and DockerImage.  ImageStreamTag references
+  // can only reference a tag within this same ImageStream.
```
Comment (soltysh) [inline image/v1/generated.proto]: Thanks @bparees I owe you one :)

**Pull request merged**